services/proctoring-service/src/controllers/proctoring.controller.js

```
@@ -72,9 +72,81 @@ async function getStudentsInExam(req, res) {
+  } catch (error) {
+    console.error('Error in getStudentsInExam controller:', error);
+    res.status(500).json({ message: 'Lỗi khi lấy sinh viên.' });
+  }
+}
+
+async function startProctoringSession(req, res) {
+  try {
+    // Lấy userId và examId (chính là submissionId) từ body của request
+    const { userId, examId } = req.body;
+    if (!userId || !examId) {
+      return res.status(400).json({ message: 'userId và examId là bắt buộc.' });
```

Biome (2.1.2)

[error] Shouldn't redeclare 'startProctoringSession'. Consider to delete it or rename it.

'startProctoringSession' is defined here:


(lint/suspicious/noRedeclare)